src/tests/Common/wasm-test-runner/WasmTestRunner.proj

```diff
@@ -1,13 +1,16 @@
-<Project Sdk="Microsoft.Build.NoTargets" DefaultTargets="BuildApp">
+<Project Sdk="Microsoft.Build.NoTargets/1.0.53" DefaultTargets="BuildApp">
```

Comment:
This should be able to just be `<Project Sdk="Microsoft.Build.NoTargets" DefaultTargets="BuildApp" />`.

It should pull the version of the SDK from the global.json at the root of the repo (and you shouldn't need the package reference below).

What issue were you hitting that led to doing it this way?

Reply:
This .proj is used to assemble the wasm apps, and that is done on the helix machines. There is no global.json there, which is why I think using NoTargets was not working. I haven't verified weather this resolves the problem though.

Reply:
I still suspect that the problem is caused by the fact that Nathan runs this msbuild script on the Helix machine; AFAIK the only reason for this is the availability of the WasmAppBuilder msbuild task. I originally talked him into refactoring the builder as a console app and not running msbuild on the Helix client but he later put it back after Santi showed him how to make msbuild available in Helix to avoid the duplication between the WasmAppBuilder msbuild task and app.

Reply:
I can, however, get rid of the pacakge reference. It doesn't need to be there; I was just trying stuff.

Reply:
Ok that makes sense. Can we add a comment explaining why we need the explicit version so that people who look at this in the future know why it's that way?

Reply:
Done.